client/scss/components/_coalition-table.scss

```diff
@@ -0,0 +1,14 @@
+.coalition-table {
+  width: 100%;
+  margin-bottom: 2em;
+  font-size: 0.9em;
+}
+
+.non-viable-coalition {
```

Comment:
could use BEM here. e.g `.coalition-table__viable` and `.coalition-table__not-viable`